bin/prepare-release.sh

```diff
@@ -14,16 +14,19 @@
 #   bin/prepare-release.sh v1.2.3
 #
 
+LANSUITE_GIT_URL="https://github.com/lansuite/lansuite.git"
+OUTPUT_DIR="/builds/"
+
 # Our own sha256sum function call.
 # That takes into account machines that don't have sha256sum but do have sha2 (i.e. macOS)
```

Comment:
I am a bit confused here.
I understand that this is intended to run within the container.
Why should we then care about the OS when we have it in our hands to satisfy the requirement by including e.g:
`RUN apt-get update && \
    apt-get install -y libdigest-sha-perl`

Or is this intended to be also callable natively on the host system?

Reply:
This is a good catch. It is meant to run inside the container.
In https://github.com/lansuite/lansuite/pull/663/commits/c922f4df7db63c87b50edaee3d35e66dcffe0032 I removed the OS independent version of the checksum function.